src/config/lang/english.go

```diff
@@ -46,6 +46,7 @@ const (
 		"using a declarative packaging strategy to support DevSecOps in offline and semi-connected environments."
 
 	RootCmdFlagLogLevel              = "Log level when running Zarf. Valid options are: warn, info, debug, trace"
+	RootCmdFlagLogFormat             = "Select a logging format. Defaults to 'text'. Valid options are: 'text', 'json'"
```

Comment:
We are working to remove the lang package so we shouldn't add more to it.

Reply:
My intent was to keep using the existing structure until we moved it all at once, but there's not rly a downside to adding the string directly in root.go. Moved it over it latest.